troposphere/firehose.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2014, Guillem Anguera <ganguera@gmail.com>
```

Comment:
should i chop the whole thing? I personally dont care to have my info here, its in the commit logs if anyone cares.